src/colorpickr.js

```diff
@@ -27,38 +25,62 @@ import {
   isDark
 } from './colorfunc';
 
-const isRGBMode = c => c === 'r' || c === 'g' || c === 'b';
-const isHSLMode = c => c === 'h' || c === 's' || c === 'l';
+const isRGBMode = (c) => c === 'r' || c === 'g' || c === 'b';
```

Comment:
if you made a `Set(["r","g","b"])` in module scope, then did a has check, this would be a more efficient check.